src/main/kotlin/com/atlassian/performance/tools/infrastructure/Pidstat.kt

```diff
@@ -0,0 +1,35 @@
+package com.atlassian.performance.tools.infrastructure
+
+import com.atlassian.performance.tools.infrastructure.api.os.MonitoringProcess
+import com.atlassian.performance.tools.infrastructure.api.os.OsMetric
+import com.atlassian.performance.tools.infrastructure.api.process.RemoteMonitoringProcess
+import com.atlassian.performance.tools.ssh.api.SshConnection
+
+internal class Pidstat private constructor(
+    private val logPath: String,
+    private val pidstatParams: String
+) : OsMetric {
+    override fun startMonitoring(
+        connection: SshConnection
+    ): MonitoringProcess {
```

Comment:
Consistent with other methods:
```suggestion
    override fun startMonitoring(connection: SshConnection): MonitoringProcess {
```

Reply:
That's exactly how you make it inconsistent :)

Reply:
WDYM? All other methods in this file don't chop single-param methods.

Reply:
I've changed `Iostat` and `Vmstat` to make all consistent to the rest of the project